git/git_intro.qmd

```diff
@@ -1,8 +1,16 @@
 # Introduction to Git
 
-## What is git
+## What is Git
 
-## What should git be used for
+Git is nowadays the most widely used distributed version control system, especially in software development. By opposition to centralized version control systems, with Git, the code source, including its full history, is mirrored on every developer's computer.
+
+Git is the most popular tool, even though it might not be the most user-friendly one. It has a lot of options/commands and a specific jargon. Fortunately there are many "Git cheat sheets" (such as https://education.github.com/git-cheat-sheet-education.pdf).
```

Comment:
```suggestion
Git is the most popular tool, even though it might not be the most user-friendly one. It has a lot of options/commands and specific jargon. Fortunately there are many "Git cheat sheets" (such as https://education.github.com/git-cheat-sheet-education.pdf).
```